modules/scheduler/scheduler/src/main/java/org/motechproject/scheduler/service/impl/MotechScheduledJob.java

```diff
@@ -45,7 +47,8 @@ public void execute(JobExecutionContext jobExecutionContext) {
             String eventType = jobDataMap.getString(SchedulerConstants.EVENT_TYPE_KEY_NAME);
             Map<String, Object> params = jobDataMap.getWrappedMap();
             params.remove(SchedulerConstants.EVENT_TYPE_KEY_NAME);
-            params.put("JobID", jobId);
+            params.put(SchedulerConstants.JOB_ID_KEY, jobId);
+            params.put(TasksEventParser.CUSTOM_PARSER_EVENT_KEY, SchedulerEventParser.PARSER_NAME);
```

Comment:
I don't like the idea of putting this into all job executions, any way to limit this?